prusti-viper/src/encoder/encoder.rs

```diff
@@ -715,15 +714,15 @@ impl<'v, 'tcx> Encoder<'v, 'tcx> {
                 let bits = scalar_value.to_u32().unwrap();
                 vir::Expr::Const(
                     vir::ConstExpr {
-                        value: vir::Const::Float(vir::FloatConst::F32(bits)),
+                        value: vir::Const::Float(FloatConst::F32(bits)),
```

Comment:
This should be (also in other places):
```suggestion
                        value: vir::Const::Float(vir::FloatConst::F32(bits)),
```